scalding-core/src/test/scala/com/twitter/scalding/TypedFieldsTest.scala

```diff
@@ -44,20 +44,20 @@ class TypedFieldsTest extends WordSpec with Matchers {
           outMap("bar") shouldBe 6
         }
         .run
-        .finish
+        .finish()
```

Comment:
PS: i'm actually not even sure we need `finish` anymore. That was just a type trick that I think is not needed with Scalatest. ? Can to try just removing a few and seeing that it still compiles and tests?